src/Host/Client/Impl/Host/LocalRHostConnector.cs

```diff
@@ -91,8 +92,11 @@ public LocalRHostConnector(string name, string rHome, string rhostDirectory = nu
                     process = Process.Start(psi);
                     process.EnableRaisingEvents = true;
 
+                    //GeneralLog.Write($"Broker start {process.Id}");
```

Comment:
I have changes in this file, so it will be a bit easier to merge :)

---
In reply to: [77022054](https://github.com/Microsoft/RTVS/pull/2236#discussion_r77022054) [](ancestors = 77022054)